### PR TITLE
MSR - Fix area name typos in Patches tab

### PR DESCRIPTION
**Pull request by dyceron**

### CODE REVIEW

File: randovania/games/samus_returns/gui/ui_files/preset_msr_patches.ui [hunk 1]
```diff
@@ -143,7 +143,7 @@
            <item>
             <widget class="QCheckBox" name="surface_crumbles_check">
              <property name="text">
-              <string>Crumble Blocks in Surface - Cavern Cavity</string>
+              <string>Crumble Blocks in Surface East - Cavern Cavity</string>
```
Comment (Miepee) [inline randovania/games/samus_returns/gui/ui_files/preset_msr_patches.ui:146]: I thought we intentionally left out the "East" here?

File: randovania/games/samus_returns/gui/ui_files/preset_msr_patches.ui [hunk 2]
```diff
@@ -133,7 +133,7 @@
...
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Removes the Grapple Blocks that are near the exit elevators of certain areas. These include:&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Area 4 Central Caves to Area 4 Crystal Mines&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Area 5 Tower Lobby to Area 6 &lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Area 6 to Area 7 &lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Area 7 to Area 8 &lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (Miepee) [inline randovania/games/samus_returns/gui/ui_files/preset_msr_patches.ui:136]: You remvoed some minuses here, was that intentional?
```diff
- Area 4 Central Caves to Area 4 - Crystal Mines
+ Area 4 Central Caves to Area 4 Crystal Mines

- Area 5 - Tower Lobby to Area 6 
+ Area 5 Tower Lobby to Area 6

File: randovania/games/samus_returns/gui/ui_files/preset_msr_patches.ui (hunk 1 above)
Comment (dyceron) [reply randovania/games/samus_returns/gui/ui_files/preset_msr_patches.ui:146]: Likely not. Plus, that's not helpful if someone were to open the DB to find the room.

File: randovania/games/samus_returns/gui/ui_files/preset_msr_patches.ui (hunk 2 above)
Comment (dyceron) [reply randovania/games/samus_returns/gui/ui_files/preset_msr_patches.ui:136]: We don't have hyphens in the area names.

Comment (Miepee) [reply randovania/games/samus_returns/gui/ui_files/preset_msr_patches.ui:136]: Oh i see, these aren't room names.

**Review by Miepee — CHANGES_REQUESTED** [review]
missing changelog entry

File: CHANGELOG.md [hunk 3]
```diff
@@ -98,6 +98,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed: If progressives are enabled, the models for Wave Beam and High Jump Boots now face right to be more recognizable.
 - Fixed: Beam Doors are no longer mismatched with the door they are attached to, which would prevent certain doors from being opened.
 - Fixed: Crash when defeating the Larva Metroids in reverse.
+- Fixed: Typos in `Patches` tab regarding area names
```
Comment (Miepee) [inline CHANGELOG.md:101]:
```suggestion
- Fixed: Typos in `Patches` tab regarding area names.
```

**Review by Miepee — APPROVED** [review]

**Pull request merged**